python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -457,42 +482,71 @@ def makePlot(
+            for ax, zoomFactor, histColor in zip(skyprojAxes, [1.0, *zoomFactors], histColors):
                 extent = getZoomedExtent(fullExtent, zoomFactor)
 
                 sp = skyproj.GnomonicSkyproj(
                     ax=ax,
                     lon_0=ctr_lon,
                     lat_0=ctr_lat,
                     extent=extent,
-                    **plotConfig.projectionKwargs,
+                    **projectionKwargs,
                 )
-                if "cmap" not in plotConfig.colorbarKwargs.keys() or plotConfig.publicationStyle:
-                    plotKwargs = {"cmap": "viridis"}
-                else:
-                    plotKwargs = {}
-                for key in plotConfig.colorbarKwargs.keys():
-                    if key != "cmap" and not plotConfig.publicationStyle:
-                        plotKwargs[key] = plotConfig.colorbarKwargs[key]
-                if "cmap" not in plotKwargs.keys():
-                    plotKwargs["cmap"] = "viridis"
 
-                sp.draw_hspmap(mapData, zoom=False, cmap=plotKwargs["cmap"])
+                sp.draw_hspmap(mapData, zoom=False, cmap=colorbarKwargs["cmap"], rasterized=rasterized)
+
+                sp.ax.set_xlabel("R.A.", labelpad=labelpad, fontsize=rcparams["axes.labelsize"])
+                sp.ax.set_ylabel("Dec.", labelpad=labelpad, fontsize=rcparams["axes.labelsize"])
+
+                # Specify the size and padding of the colorbar axes with
+                # respect to the main axes.
+                refAx = AxesX(ax) if colorbarLocation in ("left", "right") else AxesY(ax)
+                cbsize = Fraction(colorBarFraction, refAx)
+                cbpad = Fraction(colorBarPad, refAx)
+
+                # Make divider and a colorbar axes to be attached to the main
+                # axes.
+                divider = make_axes_locatable(ax)
+                cax = divider.append_axes(colorbarLocation, size=cbsize, pad=cbpad)
 
-                sp.ax.set_xlabel("RA")
-                sp.ax.set_ylabel("Dec")
                 cbar = sp.draw_colorbar(
                     **{
-                        "location": "right",
+                        "location": colorbarLocation,
                         "aspect": colorBarAspect,
-                        "fraction": 0.15,
-                        "pad": 0,
-                        **plotKwargs,
+                        "fraction": colorBarFraction,
+                        "pad": colorBarPad,
+                        "cax": cax,
+                        **colorbarKwargs,
                     }
                 )
                 cbar.ax.tick_params(labelsize=colorbarTickLabelSize)
-                cbarText = "Full Tract" if zoomFactor == 1.0 else f"{self.prettyPrintFloat(zoomFactor)}x Zoom"
+                if plotConfig.publicationStyle:
+                    if plotInfo["unit"] not in ["dimensionless", "N/A"]:
+                        cbarText = f"{plotInfo['property']} / {plotInfo['unit']}"
```

Comment:
Please follow the guidelines on units and styles.

Reply:
This is the Monthly Notices style where the quantities on the axes are dimensionless because they've been divided by the unit, but we had a long Slack discussion on this last year where it was decided that `/` should not be used 😄